### PR TITLE
More specific targeting of rich text selectors

### PR DESCRIPTION
**Pull request by drzax**

### CODE REVIEW

**Review by AshKyd — APPROVED** [review]
First glance I thought this might affect my PR, but it doesn't seem to. Looks good :)

File: src/app/components/utilities/richtext.scss [hunk 1]
```diff
@@ -412,7 +413,7 @@
       }
     }
 
-    // TODO: This is a temporary fix for a but in PL's dark mode implementation
+    // TODO: This is a temporary fix for a but in PL's dark mode
```
Comment (jtfell) [inline src/app/components/utilities/richtext.scss]: 👀  fix that but

Comment (AshKyd) [reply src/app/components/utilities/richtext.scss]: 🍑 ✅

Comment (drzax) [reply src/app/components/utilities/richtext.scss]: 😆

**Pull request merged**